### PR TITLE
GDS Configuration Documentation

### PR DESCRIPTION
**Pull request by bbengfort**

### CODE REVIEW

File: docs/content/gds/admin/configuration/gds.en.md [hunk 1]
```diff
@@ -1,7 +1,159 @@
...
+| GDS_ADMIN_ALLOW_ORIGINS                  | []string          | http://localhost,http://localhost:3000 | A list of allowed origins for the CORS middleware to accept.                                           |
+| GDS_ADMIN_COOKIE_DOMAIN                  | string            |                                        | The domain to set secure cookies for (particularly for CSRF and authentication)                        |
+| GDS_ADMIN_AUDIENCE                       | string            |                                        | The audience to set and verify in JWT tokens issued by the Admin API                                   |
+| GDS_ADMIN_OAUTH_GOOGLE_AUDIENCE          | string            |                                        | The audience from the Google OAuth config to verify Google login tokens                                |
+| GDS_ADMIN_OAUTH_AUTHORIZED_EMAIL_DOMAINS | []string          |                                        | The list of authorized email domains to allow access to the admin UI for (e.g. trisa.io)               |
```
Comment (daniellemaxwell) [inline docs/content/gds/admin/configuration/gds.en.md:43]:
```suggestion
| GDS_ADMIN_OAUTH_AUTHORIZED_EMAIL_DOMAINS | []string          |                                        | The list of authorized email domains to allow access to the admin UI (e.g. trisa.io)               |
```

File: docs/content/gds/admin/configuration/gds.en.md [hunk 2]
```diff
@@ -1,7 +1,159 @@
...
+| GDS_ADMIN_TOKEN_KEYS                     | map[string]string |                                        | A mapping of key id (ksuid/ulid) to the path to an RSA signing key in PEM format for JWT token signing |
+
+Note that the enabled flag is only respected if `GDS_MAINTENANCE` is `false`, otherwise maintenance mode supersedes service enabled flags.
+
+### Members API
+
+The Members API is a gRPC API that is secured by TRISA verified mTLS and is configured as follows:
```
Comment (daniellemaxwell) [inline docs/content/gds/admin/configuration/gds.en.md:50]:
```suggestion
The Members API is a gRPC API that is secured by TRISA, verified by mTLS, and is configured as follows:
```

File: docs/content/gds/admin/configuration/gds.en.md [hunk 3]
```diff
@@ -1,7 +1,159 @@
...
+
+Do not set the `SECTIGO_ENDPOINT` unless you're in testing, development, or staging mode and are pointing it to a local cathy server.
+
+### Backups
+
+The Backup manager is only available when the GDS is using the LevelDB database store. It is a background process that clones the database into a zipped folder on disk that can be downloaded for backup purposes.
+
+| EnvVar              | Type     | Default | Description                                                      |
+|---------------------|----------|---------|------------------------------------------------------------------|
+| GDS_BACKUP_ENABLED  | bool     | false   | If true, enables the backup background process.                  |
```
Comment (daniellemaxwell) [inline docs/content/gds/admin/configuration/gds.en.md:126]: The description sentence here ends in a `.` while others don't. This is a consistency nitpick and may be ignored.

File: docs/content/gds/admin/configuration/gds.en.md [hunk 4]
```diff
@@ -1,7 +1,159 @@
...
+### Sentry
+
+The GDS uses [Sentry](https://sentry.io/) to assist with error monitoring and performance tracing. Configure GDS to use Sentry as follows:
+
+| EnvVar                       | Type    | Default     | Description                                                                                       |
+|------------------------------|---------|-------------|---------------------------------------------------------------------------------------------------|
+| GDS_SENTRY_DSN               | string  |             | The DSN for the Sentry project. If not set then Sentry is considered disabled.                    |
+| GDS_SENTRY_SERVER_NAME       | string  |             | Optional - a server name to tag Sentry events with.                                               |
+| GDS_SENTRY_ENVIRONMENT       | string  |             | The environment to report (e.g. development, staging, production). Required if Sentry is enabled. |
+| GDS_SENTRY_RELEASE           | string  | {{version}} | Specify the release of Ensign for Sentry tracking. By default this will be the package version.   |
```
Comment (daniellemaxwell) [inline docs/content/gds/admin/configuration/gds.en.md:154]: Should `Ensign` be in the description?

File: docs/content/gds/admin/configuration/gds.en.md [hunk 5]
```diff
@@ -1,7 +1,159 @@
...
+
+SendGrid is considered **enabled** if the SendGrid API Key is set. The service and admin email addresses are required if SendGrid is enabled.
+
+### CertMan
+
+The CertMan is a background process that runs on the GDS and manages certificate requests, interacts with Sectigo, and finalizes the VASP registration process. It is configured as follows:
+
+| EnvVar                          | Type     | Default           | Description                                                                                                    |
+|---------------------------------|----------|-------------------|----------------------------------------------------------------------------------------------------------------|
+| GDS_CERTMAN_ENABLED             | bool     | true              | If false, disables the CertMan background process.                                                             |
```
Comment (daniellemaxwell) [inline docs/content/gds/admin/configuration/gds.en.md:103]: The description sentence here ends in a `.` while others don't. This is a consistency nitpick and may be ignored.

File: docs/content/gds/admin/configuration/gds.en.md [hunk 6]
```diff
@@ -1,7 +1,159 @@
...
+
+When connecting to a LevelDB, the path to the directory on disk where the leveldb should be stored must be passed to the DSN. To specify a relative path, use three slashes: `leveldb:///relpath/to/db`, to specify an absolute path use four: `leveldb:////abspath/to/db`.
+
+### Email and SendGrid
+
+The GDS uses [SendGrid](https://sendgrid.com/) to send email notifications and to enable communication workflows with users and admins. Configure GDS to use SendGrid as follows:
+
+| EnvVar                 | Type   | Default                                | Description                                                                                 |
+|------------------------|--------|----------------------------------------|---------------------------------------------------------------------------------------------|
+| GDS_SERVICE_EMAIL      | string | TRISA Directory Service                | The email address used as the sender for all emails from the GDS system.                    |
```
Comment (daniellemaxwell) [inline docs/content/gds/admin/configuration/gds.en.md:86]: Some of the description sentences end in a `.` while others don't. This is a consistency nitpick and may be ignored.

**Review by daniellemaxwell — APPROVED** [review]
This is thorough and looks good!

File: docs/content/gds/admin/configuration/gds.en.md (hunk 4 above)
Comment (bbengfort) [reply docs/content/gds/admin/configuration/gds.en.md:154]: Ooops - good catch! I'll fix it in the next PR.

**Pull request merged**